### PR TITLE
Bump rusqlite version

### PR DESCRIPTION
**Pull request by lavrd**
Bump rusqlite version to be able to use latest one.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,6 @@
 Refinery.toml
 target/
 Cargo.lock
+
+# IDEs files
+.vscode
```
Comment (jxs) [inline .gitignore:14]:
```suggestion

```

**Review by jxs — APPROVED** [review]

**Review by jxs — APPROVED** [review]
thanks!

**Comment by jxs** [comment]
will wait on #296 to then cut a release ok?

**Comment by lavrd** [comment]
> will wait on #296 to then cut a release ok?

Of course, not a problem :)

**Pull request merged**